build/dockerfiles/KServe/entrypoint.sh

```diff
@@ -6,8 +6,6 @@ echo "Binary version info:"
 kit version
 
 read -r -a UNPACK_FLAGS <<< "$KIT_UNPACK_FLAGS"
-read -r AWS_ECR_REGION <<< "$AWS_ECR_REGION"
-read -r AWS_ROLE_ARN <<< "$AWS_ROLE_ARN"
```

Comment:
I think these changes were dropped in the rebase -- the AWS IRSA handling is still useful outside the more general `login` use case.

Reply:
Ah I missed the change below, sorry